manifests/init.pp

```diff
@@ -18,6 +18,7 @@
   $rails_env                = $puma::params::rails_env,
   $rvm_ruby                 = $puma::params::rvm_ruby,
   $service_type             = $puma::params::service_type,
-  $restart_command          = $puma::params::restart_command
+  $restart_command          = $puma::params::restart_command,
+  $bunlder_path             = $puma::params::bundler_path,
```

Comment:
typo?  Looks like it should be $bundler_path

Reply:
I thought I had fixed this typo.  Let me fix it again :)